src/gui/components/components.hpp

```diff
@@ -57,7 +57,7 @@ namespace big
 		template<template_str cmd_str, ImVec2 size = ImVec2(0, 0), ImVec4 color = ImVec4(0.24f, 0.23f, 0.29f, 1.00f)>
 		static void player_command_button(player_ptr player = g_player_service->get_selected(), const std::vector<uint64_t> args = {}, std::optional<const std::string_view> label_override = std::nullopt)
 		{
-			static player_command* command = dynamic_cast<player_command*>(command::get(rage::joaat(cmd_str.value)));
+			static player_command* command = player_command::get(rage::joaat(cmd_str.value));
```

Comment:
how it got compiled. It should show error`cannot convert from 'big::command *' to 'big::player_command *'`

Reply:
correct.

Reply:
LGTM